bittensor/axon.py

```diff
@@ -318,7 +320,10 @@ def attach(
 
         # Add the endpoint to the router, making it available on both GET and POST methods
         self.router.add_api_route(
-            f"/{request_name}", forward_fn, methods=["GET", "POST"]
+            f"/{request_name}",
+            forward_fn,
+            methods=["GET", "POST"],
+            dependencies=[Depends(self.verify_body_integrity)],
```

Comment:
Really clean design! The api route now depends on the `verify_body_integrity`?

Reply:
Yessir! Seems to be pretty fast in my initial testing as well.